frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -790,6 +800,72 @@ dbg:guard(FileManagerMenu, 'setUpdateItemTable',
         end
     end)
 
+function FileManagerMenu:moveBookMetadata()
+    local DocSettings = require("docsettings")
+    local FileChooser = self.ui.file_chooser
+    local function scanPath()
+        local sys_folders = { -- do not scan sys_folders
+            ["/dev"]  = true,
+            ["/proc"] = true,
+            ["/sys"]  = true,
+        }
```

Comment:
Dunno if that lists enough of the directories to avoid, and some devices may have others?
I was going to suggest to avoid any top level directories anchored in the root `/`, so any fullpath with a single component.
Which might be ok on Kobo, Kindle, Android... but on the emulator, one may have his books in a `/books/`.